Remotion/Data/DomainObjects/Queries/Configuration/QueryDefinition.cs

```diff
@@ -111,14 +114,18 @@ public QueryDefinition (
         && !collectionType.Equals(typeof(DomainObjectCollection))
         && !collectionType.IsSubclassOf(typeof(DomainObjectCollection)))
     {
-      throw new ArgumentException(string.Format(
-          "The collectionType of query '{0}' must be 'Remotion.Data.DomainObjects.DomainObjectCollection' or derived from it.", queryID), nameof(collectionType));
+      throw new ArgumentException(
+          string.Format(
+              "The collectionType of query '{0}' must be 'Remotion.Data.DomainObjects.DomainObjectCollection' or derived from it.",
+              queryID),
+          nameof(collectionType));
```

Comment:
strictly speaking this is a change unrelated to the rest and considered noise. if you want to keep it, it would be great if you could split the commits so one does the various cleanups (here and in the other lines in this file) and the other the changes.